source/atemplates/sidebar.html

```diff
@@ -23,7 +23,7 @@ <h3>Get Cyclus</h3>
     Current version: <b>1.6.0</b>
     <br><br>
     Install:
-    <div class="highlight-bash" style="width:97.5%"><div class="highlight"><pre><span class="nv">$ </span>conda install cyclus cycamore</pre></div></div>
+    <div class="highlight-bash" style="width:97.5%"><div class="highlight"><pre><span class="nv">$ </span>conda config --add channels conda-forge<br><span class="nv">$ </span>conda install cyclus cycamore</pre></div></div>
```

Comment:
This is a little simpler, maybe?
```suggestion
    <div class="highlight-bash" style="width:97.5%"><div class="highlight"><span class="nv">$ </span>conda install -c conda-forge cyclus cycamore</pre></div></div>
```

Reply:
I super agree that this is simpler. I didn't know you could combine the commands like that! I'll make the change. You forgot an opening <pre> tag, though, so I'll make the commit from my machine to make sure that nice little white box shows up around the command (otherwise it would just display as text on the yellow sidebar background and look funny).